### PR TITLE
SOUTH AFRICA_ Alexander_Msumba

### PR DESCRIPTION
**Pull request by MSUMBA-art**
# Your Details

- Your Name: Alexander Msumba, Jr
- Your City: Cape Town
- Your Slack Name: Alexander Msumba

# Homework Details: Github Fix Errors

- Module: Github
- Week: 2

### CODE REVIEW

**Comment by mulindierick** [comment]
Hi Alexander, Great job. 
Just wanted to say that the opposite of black is white, not red. You might want to check that on question two.

File: homework.md [hunk 1]
```diff
@@ -4,110 +4,110 @@
...
 
-You send the changes of your local repository to Github
+Push means to submit commits from your repository to GitHub 
 
 ## Question 14
 
 What is Github?
 
-A place for us to store and share our code.
+A web service where you can sign up and get and account
```
Comment (LawrenceBaatjies) [inline homework.md:89]: I might have worded it as  " _A web service based distribution control system where you can sign up and get and account_."

File: homework.md [hunk 2]
```diff
@@ -4,110 +4,110 @@
...
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+Red
```
Comment (LawrenceBaatjies) [inline homework.md:13]: The Opposite of Black.. Would Be white.

**Comment by JarrodBen** [comment]
With Question 9, I would try and be a bit more descriptive of what it means to fork a repository. You're in the right direction, but to where do you "copy" it to?

**Review by BrettScheepers — COMMENTED** [review]
The updated version looks good

**Review by JarrodBen — APPROVED** [review]

File: homework.md [hunk 3]
```diff
@@ -4,110 +4,110 @@
...
+Help to share information
+Enable people to make their own changes
+Allow us to make checkpoints so that we can save our work as we go along
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You copy it
```
[the remark this answers is not in the document]
Comment (JarrodBen) [reply homework.md:62]: Awesome, it's describes the functionality of what it means to `fork` a repository.

**Review by MSUMBA-art — COMMENTED** [review]
thanks guys for your reviews

File: homework.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (JarrodBen) [reply homework.md:62]: I am done with my review, all looks good Alexander

**Review by JarrodBen — COMMENTED** [review]
Awesome, it's describes the functionality of what it means to `fork` a repository.

**Pull request closed without merging**